prow/flagutil/github.go

```diff
@@ -22,20 +22,31 @@ import (
 	"net/url"
 
 	"github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
 
+	"k8s.io/test-infra/pkg/ghclient"
 	"k8s.io/test-infra/prow/config/secret"
 	"k8s.io/test-infra/prow/git"
 	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/githuboauth"
 )
 
 // GitHubOptions holds options for interacting with GitHub.
 type GitHubOptions struct {
+	host                string
 	endpoint            Strings
 	graphqlEndpoint     string
 	TokenPath           string
 	deprecatedTokenFile string
 }
 
+// DefaultGitHubOptions is the GitHubOptions with default values.
+var DefaultGitHubOptions GitHubOptions = GitHubOptions{
```

Comment:
To Reviewers:

This is required in one of the tests as it relies on GitHubOptions, and there is no good way to instantiate one of GitHubOptions instances from other packages since all the fields here are private.

Reply:
Maybe `NewGitHubOptions() GitHubOptions` is better.